nvm.sh

```diff
@@ -1303,6 +1320,30 @@ nvm_die_on_prefix() {
   fi
 }
 
+# Succeeds if $IOJS_VERSION represents an io.js version that has a
+# Solaris binary, fails otherwise.
+# Currently, only io.js 3.3.1 has a Solaris binary available, and it's the
+# latest io.js version available. The expectation is that any potential io.js
+# version later than v3.3.1 will also have Solaris binaries.
+iojs_version_has_solaris_binary() {
+  local IOJS_VERSION=$1
```

Comment:
This is great - could you also add lines that error out with an explicit error code if it's not an iojs version? Feel free to optimize a bit by storing `"$(nvm_strip_iojs_prefix $IOJS_VERSION)"` in a local, and just testing if it's equal to `$IOJS_VERSION` :-)